notebooks/274-Real-ESRGAN-image-restoration/274-Real-ESRGAN-image-restoration.ipynb

```diff
@@ -0,0 +1,740 @@
+{
```

Comment:
<p>&gt;Similar to ESRGAN, the modle adopted DIV2K, Flickr2K and OutdoorSceneTraining datasets for training. The training HR patch size is set to 256. Models are trained with four NVIDIA V100 GPUs with a total batch size of 48. Adam optimizer employed.</p><p> </p><p>model -&gt; model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/939/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>